src/me/StevenLawson/TotalFreedomMod/Commands/Command_report.java

```diff
@@ -40,7 +40,7 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
 
         if (TFM_AdminList.isSuperAdmin(player))
         {
-            playerMsg(ChatColor.RED + "You can not report an admin.");
+            playerMsg(ChatColor.RED + "You may report admin abuse on the forums, however, please include screenshots!  https://pravi.us/vfa");
```

Comment:
@zD12 I have this feeling that this should be two sentences, split between 'forums' and 'however'.